net/strongswan/patches/305-minimal_dh_plugin.patch

```diff
@@ -68,7 +68,7 @@
 +	../gmp/gmp_diffie_hellman.c ../gmp/gmp_diffie_hellman.h 
 +
 +	
-+libstrongswan_gmpdh_la_LDFLAGS = -module -avoid-version -Wl,-Bstatic -Wl,-lgmp -Wl,-Bdynamic -Wl,--as-needed
++libstrongswan_gmpdh_la_LDFLAGS = -module -avoid-version -Wl,-Bstatic -Wl,-lgmp -Wl,-Bdynamic -Wl,--as-needed $(FPIC)
```

Comment:
```suggestion
+libstrongswan_gmpdh_la_LDFLAGS = -module -avoid-version -Wl,-Bstatic -Wl,-lgmp -Wl,-Bdynamic -Wl,--as-needed "$(FPIC)"
```

Isn't that saver?

Reply:
No, because spaces between args need to be maintained.

Reply:
I'd really rather have all of $(TARGET_CFLAGS) and $(TARGET_LDFLAGS) get interpolated...

Reply:
Ack